app/controllers/v2/test_results_controller.rb

```diff
@@ -19,6 +19,12 @@ def os_versions
     permission_for_action :os_versions, Rbac::SYSTEM_READ
     permitted_params_for_action :os_versions, { filter: ParamType.string }
 
+    def ssg_versions
```

Comment:
SSG is something from the old terminology, it should be `security_guide_versions`